src/Peachpie.Runtime/Operators.cs

```diff
@@ -657,6 +717,55 @@ public static PhpAlias EnsureItemAlias(this IPhpArray array, PhpValue index, boo
         /// </summary>
         public static PhpValue GetItemValue(PhpValue value, PhpValue index, bool quiet = false) => value.GetArrayItem(index, quiet);
 
+        public static bool TryGetItemValue(this PhpArray value, string index, out PhpValue item)
+        {
+            if (value != null && value.TryGetValue(index, out item) && IsSet(item))
+            {
+                return true;
+            }
+            else
+            {
+                item = default;
+                return false;
+            }
+        }
+
+        public static bool TryGetItemValue(this PhpArray value, PhpValue index, out PhpValue item)
+        {
+            if (value != null && index.TryToIntStringKey(out var key) && value.TryGetValue(key, out item) && IsSet(item))
+            {
+                return true;
+            }
+            else
+            {
+                item = default;
+                return false;
+            }
+        }
+
+        public static bool TryGetItemValue(this PhpValue value, PhpValue index, out PhpValue item)
+        {
+            if (value.Object is PhpArray array)
```

Comment:
`value.Object is PhpAlias alias` ?

Reply:
Changed to `IsPhpArray`, which handles that, in e8f7755671f70fa15bd3f069454a58a54ffb6d0a.